app/org/sagebionetworks/bridge/dynamodb/DynamoTask.java

```diff
@@ -56,111 +68,149 @@ public TaskStatus getStatus() {
             return TaskStatus.FINISHED;
         } else if (startedOn != null) {
             return TaskStatus.STARTED;
-        } else if (expiresOn != null && DateTime.now().isAfter(expiresOn)) {
-            return TaskStatus.EXPIRED;
-        } else if (scheduledOn != null && DateTime.now().isBefore(scheduledOn)) {
-            return TaskStatus.SCHEDULED;
+        }
+        if (timeZone != null) {
+            DateTime now = DateTime.now(timeZone);
+            if (localExpiresOn != null && now.isAfter(getExpiresOn())) {
+                return TaskStatus.EXPIRED;
+            } else if (localScheduledOn != null && now.isBefore(getScheduledOn())) {
+                return TaskStatus.SCHEDULED;
+            }
         }
         return TaskStatus.AVAILABLE;
     }
+    
+    @Override
+    @DynamoDBIgnore
+    public DateTime getScheduledOn() {
+        return getInstant(getLocalScheduledOn());
+    }
+    
+    @Override
+    @DynamoDBIgnore
+    public DateTime getExpiresOn() {
+        return getInstant(getLocalExpiresOn());
+    }
+    
+    private DateTime getInstant(LocalDateTime localDateTime) {
+        return (localDateTime == null) ? null : localDateTime.toDateTime(timeZone);
+    }
+
+    /**
+     * The scheduled time without a time zone. This value is stored, but not returned in the JSON of the API. It is
+     * localized using the caller's time zone.
+     */
+    @DynamoDBAttribute
+    @DynamoDBMarshalling(marshallerClass = LocalDateTimeMarshaller.class)
     @JsonIgnore
+    public LocalDateTime getLocalScheduledOn() {
+        return localScheduledOn;
+    }
```

Comment:
I only want to expose instance times, but persist LocalDateTime. So it is never referenced anywhere else in the code and it seems confusing to expose it, it's more like an implementation detail of ensuring expiresOn and scheduledOn are set to the requested time zone.